solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.adoc

```diff
@@ -864,8 +872,15 @@ Consumer<Message<List<Payload>>> input() {
 <1> A batch of messages is really just a single Spring `Message` whose payload is a list of individual message payloads.
 <2> The `solace_scst_batchedHeaders` message header contains the consolidated list of message headers for each of the individual messages in the batch.
 
-IMPORTANT: Message batches are non-transacted. A batch that this binder creates is fundamentally a collection of individual messages and must not be treated as a single consistent unit.
+.Transacted Batch Consumers
+[TIP]
+====
+By default, message batches are non-transacted (i.e. `transacted` is set to `false`). When in this mode, a batch that this binder creates is fundamentally a collection of individual messages and must not be treated as a single consistent unit.
```

Comment:
When transacted=false, the application has a single AcknowledgmentCallback for the whole batch. Would that be considered a single consistent unit? A batch succeeds/fails as a whole. 

Or maybe 'consitent unit' means something else?

Reply:
it's just saying that each message in the batched message are just a standalone messages with no relationship between each other.

Reply:
Ok. I thought it was a statement in relation with transacted=false but it is a statement that applies to batch-mode regardless of transacted.

Reply:
yup exactly. I'll update the note to try and make that a bit clearer.

Reply:
clarified note